src/onegov/fsi/forms/subscription.py

```diff
@@ -158,31 +156,26 @@ def ensure_no_other_subscriptions(self) -> bool:
         return True
 
     def ensure_6_year_time_interval(self) -> bool:
-        if not self.request.is_admin:
```

Comment:
@BreathingFlesh Did you get rid of this condition on purpose or did you intend to move it into the common utility function and forgot?

Reply:
No, you're right. I probably wanted to move it into the utility function but forgot because the function name doesn't say anything about excepting admins.